src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -599,4 +605,40 @@ private String getElementSrcWithRetryAfterWaitForPresence(By by) {
         }
     }
 
+    /**
+     * Expands a particular panel, causing its results to load.
+     * @param panelNumber the position of panel element
+     */
+    public void loadResultPanel(int panelNumber) {
+        String panel = "panelHeading-" + panelNumber;
+        clickAutoLoadedPanelAndWaitForExpansion(panel);
+    }
+
+    /**
+     * Expands a particular section, causing its results to load.
+     */
+    public void loadResultSectionPanel(int panelNumber, int sectionNumber) {
+        String panel = "panelHeading-section-" + panelNumber + "-" + sectionNumber;
+        clickAutoLoadedPanelAndWaitForExpansion(panel);
+    }
+
+    /**
+     * Expands a particular panel, causing its results to load.
+     * @param panelNumber the position of panel element
+     */
+    public void loadResultSubmitPanel(int panelNumber) {
```

Comment:
@whipermr5 can't figure out any relevant name :sweat_smile: 
Can you suggest any ?